client/gtk/src/main.rs

```diff
@@ -33,11 +33,20 @@ pub mod scheduler;
 pub struct Glade(Arc<String>);
 
 impl Glade {
-    pub fn open<P: AsRef<Path>>(path: P) -> io::Result<Glade> {
+    pub fn open<P: AsRef<Path>>(glade_path: P) -> io::Result<Glade> {
+        let mut path = PathBuf::from(resource("glade"));
+        path.push(glade_path);
+
         let mut file = File::open(path)?;
         let mut source = String::new();
         file.read_to_string(&mut source)?;
-        Ok(Glade(Arc::new(source)))
+
+        // Replace res/* with relative link
+        let res_path = resources_path().into_os_string().into_string().unwrap();
+        let res_path = format!("{}/", res_path);
+        let glade_string = source.replace("res/", &res_path);
```

Comment:
yeah this is cursed but we're vibing with it :D

Reply:
Could you check this works for windows too?